src/amo/components/Addon/index.js

```diff
@@ -537,6 +538,11 @@ export class AddonBase extends React.Component {
                 reason={compatibility.reason}
               />
             ) : null}
+            {addon && addon.status !== 'public' ? (
```

Comment:
We have flow types for add-on status but I'm concerned about the use of a magic constant here. That said it's the only place we check status... I guess it's fine for now.

Reply:
Since that component doesn't use Flow, we should use a real constant to catch typos and protect against refactors. I would suggest doing it like this:

```diff
diff --git a/src/core/types/addons.js b/src/core/types/addons.js
index ad4abce4a..202d56c70 100644
--- a/src/core/types/addons.js
+++ b/src/core/types/addons.js
@@ -4,10 +4,12 @@ import { validAddonTypes } from 'core/constants';
 // You have to just laugh at this line!
 type AddonTypeType = validAddonTypes;
 
+export const STATUS_PUBLIC: 'public' = 'public';
+
 type AddonStatus =
   | 'beta'
   | 'lite'
-  | 'public'
+  | typeof STATUS_PUBLIC
   | 'deleted'
   | 'pending'
   | 'disabled'
```